docs/semgrep-appsec-platform/jira.md

```diff
@@ -94,56 +93,68 @@ If you opt for a drop-down or a checkbox issue type field, verify that:
 
 ### Code
 
-To create tickets for one or more Code findings, you can use the triage button:
+You can create tickets for Code findings using the **Triage** button on the:
+
+* [**Findings**](https://semgrep.dev/orgs/-/findings) page
+* Individual finding's **Details** page.
 
-* On the **Findings** page
-* On an individual finding's **Details** page.
+To create tickets:
 
-When working on the **Findings** page, you can select and create tickets for individual findings or all findings for a given rule.
+1. If you're on the [**Supply Chain > Vulnerabilities**](https://semgrep.dev/orgs/-/findings) page, select the findings for which you want tickets created; you can select and create tickets for individual findings or all findings for a given rule. Otherwise, proceed to step two.
```

Comment:
Is Supply Chain intentional? It's supposed to be Code, right?

Reply:
You're right; it was a copy and paste error from when I initially tried to combine the two sections.